v3-sdk/collecting-fees/src/example/Example.tsx

```diff
@@ -38,21 +44,21 @@ const Example = () => {
     const provider = getProvider()
     const address = getWalletAddress()
     if (!provider || !address) {
-      throw new Error('No provider or address')
+      return
```

Comment:
I was wondering why are we returning vs throwing here?

Reply:
Throwing causes an error on load when running the wallet extension example and not yet connecting the wallet - this makes it so intended use doesn't generate an error